cloudify_hostpool/rest/backend.py

```diff
@@ -125,3 +128,16 @@ def _is_alive(host):
         address, port = host['host'], host['port']
         results = scan.scan([(address, port)])
         return results[address, port]
+
+    @staticmethod
+    def _load_keyfile(host):
+        if not host['auth']:
```

Comment:
this means the keyfile will be read on every request. maybe it better to do this at the configuration level, where the check for existence already takes place anyway